internal/services/appconfiguration/client/client.go

```diff
@@ -49,73 +49,6 @@ func (c Client) LinkWorkaroundDataPlaneClientWithEndpoint(configurationStoreEndp
 	return &workaroundClient, nil
 }
 
-func (c Client) DataPlaneClient(ctx context.Context, configurationStoreId string) (*appconfiguration.BaseClient, error) {
```

Comment:
This is to remove unsed code (superseded by https://github.com/hashicorp/terraform-provider-azurerm/blob/8128b2bbbb62bae7cbe29403c7c78343e3fc984c/internal/services/appconfiguration/client/client.go#L25), related to PR https://github.com/hashicorp/terraform-provider-azurerm/pull/20082